types/err-code/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for err-code 2.0
+// Project: https://github.com/IndigoUnited/js-err-code#readme
+// Definitions by: George <https://github.com/zorji>
+//                 Cayman <https://github.com/wemeetagain>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type Scalar = Exclude<any, object>;
```

Comment:
Pretty sure this ends up just being `any` - maybe just write `number | string | boolean | bigint | null | undefined`

Reply:
Thanks for the suggestion. I have updated the code.